### DISCUSSION TITLE
Attaching videos to issues

### DISCUSSION THREAD

**Discussion by Timmmm** [community/community#23179] [New to GitHub]
<p>There should be a way to attach video clips to issues. Currently you have to zip them :nauseated_face:.</p>
<p>Ideally it would at least support h.264 in mov because that is what Apple’s handy QuickTime screen recorder makes.</p>

**Comment by wei** [comment]
<p>We’ve been using gif : )</p>

**Comment by zhyiyu** [comment]
<p>This would be a great feature, having to convert videos to gif prevents more people from sharing their thoughts.</p>

**Comment by jorisw** [comment]
<p>CloudConvert is good for this (free):<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/30017/181075467-2f4f1e73-2c34-45bf-b6ba-f50ec7712d15.png" class="site-icon" width="32" height="32">
      <a href="https://cloudconvert.com/mov-to-gif" target="_blank" rel="noopener nofollow ugc">cloudconvert.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://cloudconvert.com/mov-to-gif" target="_blank" rel="noopener nofollow ugc">MOV to GIF | CloudConvert</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by zhyiyu** [comment]
<p>I have been using it, but controlling frame rate, uploading and downloading the files and so on, is a lot of work to do…</p>

**Comment by Aidurber** [comment]
<p>I wholeheartedly agree with adding videos. My 1.6MB video is a 10MB low quality GIF with the size reduced by 50%. Adding a 50MB limit would be great if storage is an issue, but with supporting GIFs that doesn’t seem to be the case.</p>
<p>Videos are really handy for sharing a bug or an interaction on a PR.</p>

**Comment by sumit-anantwar** [comment]
<p>Embedding videos is really a must-have feature.<br>
Storage cannot be a concern, because the GIFs are usually larger than the video files.<br>
But still, If storage is a concern, we can even upload the videos to any other cloud storage like DropBox or Google Drive, and add the direct URL to the Issues/PR.<br>
What’s important is that the videos get embedded.</p>

**Comment by Timmmm** [comment, marked as answer]
<p>They actually implemented this recently (at last!). You can now attach mp4 files to Github issue comments in the same was as pictures. I didn’t test other formats.</p>

**Reply by yairEO** [reply]
Sadly it does not support `HEVEC` format

**Reply by PJUllrich** [reply]
Also, sadly it only supports 10MB files but doesn't show an error message if a user tries to upload a larger video.
